### PR TITLE
Fix google-java-format

### LINKED ISSUE #1268: gradle google-java-format tasks don't appear to work
steps to test:

- make some random change to a java file that's clearly not google formatted (like make a line longer than 100 chars)
- then run `gradlew verifyGoogleJavaFormat`

observe that no errors are displayed

### PR DESCRIPTION
**Pull request by patflynn**
For some reason the sherter google java format gradle plugin does not
seem to catch format errors in any of our modules except for the core
lib.  I tried changing the config to add all source sets and even apply
 it individually to each submodule with no luck.

 Switching to spotless seems to work as expected.

All java file changes are automatic, and now appear to match the formatting from intellij with google-java-format plugin installed.

Closes #1268

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -101,10 +102,12 @@ subprojects {
...
     }
 
-    apply plugin: "com.github.sherter.google-java-format"
+    apply plugin: 'com.diffplug.gradle.spotless'
 
-    googleJavaFormat {
-        toolVersion '1.3'
+    spotless {
+        java {
+            googleJavaFormat('1.3')
```
Comment (etanshaul) [inline build.gradle]: I noticed 1.5 is the latest? Should we use that?

**Comment by patflynn** [comment]
good idea. let me try that,.

**Comment by loosebazooka** [comment]
I vaguely remember having to skip google java format on windows. But I don't remember exactly where or how it was done.

**Comment by patflynn** [comment]
Format does appear to be different. pushed the updates.

**Comment by etanshaul** [comment]
it looks like the latest [intellij google formatter](https://plugins.jetbrains.com/plugin/8527-google-java-format) also uses 1.5, so things should line up now

**Comment by loosebazooka** [comment]
you can tell kokoro windows to skip the google java format task.

**Comment by patflynn** [comment]
Do you know why it's failing?

**Comment by etanshaul** [comment]
this may be part of it: https://github.com/google/google-java-format/issues/33

But digging through that it looks like they merged in some fixes to make things platform independent. So I'm not yet sure myself.

**Comment by patflynn** [comment]
what's odd is that my personal windows machine doesn't reproduce this error

**Comment by etanshaul** [comment]
@patflynn we can close this now right?

**Comment by patflynn** [comment]
@etanshaul done.

**Pull request closed without merging**